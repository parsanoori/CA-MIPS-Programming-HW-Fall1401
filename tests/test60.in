86
663366611
250206014
193611435
839193660
835389672
102107668
45821194
246761007
401135579
1011464012
341092623
1023173135
509644929
567437557
919267978
69347405
686220200
960280067
179449208
918870330
447816430
690793931
160350375
306061
94993015
308880374
664278907
35113023
578474317
548945156
913690508
341543479
336145252
962442888
901723494
546009587
784658717
178005694
113038894
144583882
917674454
803834729
218917162
125220313
589747499
349697819
980490843
470356099
892773187
104380134
521431024
52986235
814824085
163963220
848585875
247030854
389882828
285372570
74456948
685358161
826157250
814546118
915332349
983125443
959025991
394257712
357079595
585532341
197698649
592091845
862425113
1000785609
259975089
319002963
607191353
858230447
533174979
604842469
101684071
247313130
421783899
795890558
1042296064
1012299360
489738190
32818361