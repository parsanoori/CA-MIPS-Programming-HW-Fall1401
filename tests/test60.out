306061
32818361
35113023
45821194
52986235
69347405
74456948
94993015
101684071
102107668
104380134
113038894
125220313
144583882
160350375
163963220
178005694
179449208
193611435
197698649
218917162
246761007
247030854
247313130
250206014
259975089
285372570
308880374
319002963
336145252
341092623
341543479
349697819
357079595
389882828
394257712
401135579
421783899
447816430
470356099
489738190
509644929
521431024
533174979
546009587
548945156
567437557
578474317
585532341
589747499
592091845
604842469
607191353
663366611
664278907
685358161
686220200
690793931
784658717
795890558
803834729
814546118
814824085
826157250
835389672
839193660
848585875
858230447
862425113
892773187
901723494
913690508
915332349
917674454
918870330
919267978
959025991
960280067
962442888
980490843
983125443
1000785609
1011464012
1012299360
1023173135
1042296064